CHANGELOG.md

```diff
@@ -2,6 +2,15 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
+
+- CLI behavior change.  Commands in non-interactive mode (i.e. when `pulumi` has its output piped to
+  another process) will not default to assuming that `--yes` was passed in.  `--yes` is now
```

Comment:
```suggestion
  another process or running on CI) will not default to assuming that `--yes` was passed in.  `--yes` is now
```

something like that, so people running pulumi on CI/in scripts will think "oh, this affects us"?

Reply:
Sounds great.  have merged that in. thanks!